cli/pa_analyze_command.go

```diff
@@ -413,6 +425,118 @@ func (cmd *PaAnalyzeCmd) checkLaggingStreamReplicas(r *archive.Reader) (checkSta
 	return Pass, nil
 }
 
+const cpuUsageThreshold = 0.9 // Warn if any server is using more than 90% of the available CPU
+// checkCpuUsage checks the CPU usage of all servers and alerts if any server is using more than 90% of the available CPU
+func (cmd *PaAnalyzeCmd) checkCpuUsage(r *archive.Reader) (checkStatus, error) {
+	serverTags := r.ListServerTags()
+	examples := newCollectionOfExamples(cmd.examplesLimit)
+
+	for _, serverTag := range serverTags {
+		serverName := serverTag.Value
+		var serverVarz server.Varz
+
+		if err := r.Load(&serverVarz, &serverTag, archive.TagServerVars()); errors.Is(err, archive.ErrNoMatches) {
+			cmd.logWarning("Artifact 'VARZ' is missing for server %s", serverName)
+			continue
+		} else if err != nil {
+			return Skipped, fmt.Errorf("failed to load VARZ for server %s: %w", serverName, err)
+		}
+
+		cpuUsageForASingleCore := serverVarz.CPU / float64(serverVarz.Cores)
+
+		if cpuUsageForASingleCore > cpuUsageThreshold {
+			examples.Addf("%s: %.0f%%", serverName, serverVarz.CPU)
```

Comment:
The number you are printing here won't make much sense.

e.g. it could be 211% without knowing how many cores.

You should normalize.